packages/nimble-components/src/rich-text/components/mention-view/mention-users-view/styles.ts

```diff
@@ -0,0 +1,38 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {
+    mentionFont,
+    mentionFontColor,
+    mentionDisabledFontColor
+} from '../../../../theme-provider/design-tokens';
+import { iconAtTag } from '../../../../icons/at';
+
+export const styles = css`
+    ${display('inline-block')}
+
+    :host {
+        box-sizing: border-box;
+        font: ${mentionFont};
+        white-space: normal;
+        --ni-nimble-icon-color: ${mentionFontColor};
+    }
+
+    .control {
+        color: ${mentionFontColor};
+    }
+
+    :host([disabled]) .control {
+        color: ${mentionDisabledFontColor};
+        --ni-nimble-icon-color: ${mentionDisabledFontColor};
+    }
+
+    ${iconAtTag} {
+        position: relative;
+        right: -1.6px;
+        vertical-align: bottom;
```

Comment:
instead of this positioning can you use `host: {display :inline-flex;}` and then use flax layout for positioning?

Reply:
Since I have removed the icon here, removed all the styles related to icon.